### PR TITLE
Fix - ignore shebang lines only at the top of source files

### PR DESCRIPTION
**Pull request by lwronski**

### CODE REVIEW

**Review by romanowski — APPROVED** [review]

File: website/docs/run.md [hunk 1]
```diff
@@ -64,6 +64,19 @@ and that Scala Native only supports Linux and macOS for now.
 scala-cli my-scala-native-app/ --native
 ```
 
+## ScalaScripts
```
Comment (alexarchambault) [inline website/docs/run.md]: Why is this written as a single word?

File: website/docs/scala-scripts.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+You can define file with shebang header to self executable. It could be also run as a normal script.
+
+```bash
+cat HelloScript.sc
+# #!/usr/bin/env scala-cli
+# println("Hello world")
+
+scala-cli run HelloScript.sc
+# Hello world
+./HelloScript.sc
```
Comment (alexarchambault) [inline website/docs/scala-scripts.md]: Doesn't it need `chmod +x HelloScript.sc` too?

File: website/docs/run.md [hunk 3]
```diff
@@ -64,6 +64,19 @@ and that Scala Native only supports Linux and macOS for now.
...
+
+ScalaScripts can also be compiled and run.
```
Comment (alexarchambault) [inline website/docs/run.md]:
```suggestion
Scala CLI can also compile and run Scala scripts.
```

File: website/docs/scala-scripts.md (hunk 2 above)
Comment (lwronski) [reply website/docs/scala-scripts.md]: Yes, it need, without changing permissions the script can't be run.

File: website/docs/run.md (hunk 1 above)
Comment (lwronski) [reply website/docs/run.md]: as I think now, it should be separate words

**Comment by alexarchambault** [comment]
About the extra doc page, scripts are already mentioned [here](https://virtuslabrnd.github.io/scala-cli/docs/input#scripts). Do we really need an extra entry in the sidebar?

**Comment by lwronski** [comment]
> About the extra doc page, scripts are already mentioned [here](https://virtuslabrnd.github.io/scala-cli/docs/input#scripts). Do we really need an extra entry in the sidebar?

I didn't notice the existing section about scala script in `Input Format`.  I moved description about sheBang from `Scala Script` to `Input Format` section.

**Review by alexarchambault — APPROVED** [review]

**Pull request merged**